### PR TITLE
Feature/cleanup master

### PR DESCRIPTION
**Pull request by umulmrum**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | 
| License       | MIT

### CODE REVIEW

File: UPGRADE-7.0.md [hunk 1]
```diff
@@ -83,6 +110,25 @@ project requires WYSIWYG JavaScript, allow it by setting the configuration key
...
+Some code entities have been marked as deprecated in previous releases, but were not removed in this release because of
+wide usage, bug risks or quite a short deprecation period. They will be removed in a future Chameleon release, so be
+sure to remove calls over time.
+
+Entities that were NOT removed include (incomplete list):
+
+- Any database fields
+- Almost any method arguments
+- Classic main menu
+- Anything logging-related
```
Comment (UlrichKu) [inline UPGRADE-7.0.md]: -> "All logging-related classes and service definitions"

File: UPGRADE-7.0.md [hunk 2]
```diff
@@ -3,6 +3,33 @@ UPGRADE FROM 6.3 TO 7.0
 
 # Essentials
 
+The steps in this chapter are required to get the project up and running in version 6.3.
```
[the remark this answers is not in the document]
Comment (umulmrum) [reply UPGRADE-7.0.md]: Fixed

File: UPGRADE-7.0.md [hunk 3]
```diff
@@ -3,6 +3,33 @@ UPGRADE FROM 6.3 TO 7.0
...
+It is recommended to follow these steps in the given order.
+
+## Prepare Project
+
+Be sure to install the latest release of the Chameleon 6.3.x branch before migrating. It is recommended to remove all
+deprecated code usage from the project before migrating. This way there will be deprecation messages still available
+which will help to decide what to do with code calling deprecated entities.
+
+Note that we decided to keep some entities although this is a major release. See section
```
[the remark this answers is not in the document]
Comment (umulmrum) [reply UPGRADE-7.0.md]: lol right

File: UPGRADE-7.0.md (hunk 1 above)
Comment (umulmrum) [reply UPGRADE-7.0.md]: Also properties and methods - isn't it more or less OK as it is?

File: UPGRADE-7.0.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (umulmrum) [reply UPGRADE-7.0.md]: In a major release all old migration guides should be removed to clean things up (developers should migrate to 6.3 first and from there to 7.0, so they already followed the guide). This is also how e.g. Symfony does this.

I forgot to remove UPGRADE-6.3.md - did this now.

File: CHANGELOG-7.0.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+CHANGELOG For 7.0.x
+===================
+
+Chameleon 7.0.x is almost functionally identical to Chameleon 6.3.x. As of our backwards compatibility policy, we
+strive to make code changes in minor releases in a way that keeps backwards compatible. A major release then removes
+the compatibility layer.
+
+See the UPGRADE guide for further incompatible changes.
```
Comment (UlrichKu) [inline CHANGELOG-7.0.md]: "further" to what?

File: CHANGELOG-7.0.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+CHANGELOG For 7.0.x
+===================
+
+Chameleon 7.0.x is almost functionally identical to Chameleon 6.3.x. As of our backwards compatibility policy, we
+strive to make code changes in minor releases in a way that keeps backwards compatible. A major release then removes
```
Comment (UlrichKu) [inline CHANGELOG-7.0.md]: -> "compatibility" ?

File: CHANGELOG-7.0.md (hunk 4 above)
Comment (umulmrum) [reply CHANGELOG-7.0.md]: Rephrased the sentence.

File: CHANGELOG-7.0.md (hunk 5 above)
Comment (umulmrum) [reply CHANGELOG-7.0.md]: Fixed.

**Pull request merged**